localized/en-us/games/eggquiz/eggquizmath_levels.tsv

```diff
@@ -428,27 +427,27 @@ SpecialTest_25	1	250001, 250002, 250003, 250004, 250005
 			250004	operations_without_objects	8~9x2~9=?		horizontal			16~81				
 			250005	operations_without_objects	2~9x2~9=?		horizontal			4~81				
 SpecialTest_26	1	260001~260010, 260001~260010, 260001~260010, 260001~260010, 260001~260010												
-			260001	word_problem	ç×Úæ¢Çæ Ùð 18 Õýæ©ÙèÁ ·UUUUô Õð·UUUU ç·UUUUØæÐ ©âÙð 2 ŒÜðÅô¢ ÂÚ â×æÙ â¢wØæ ×ð¢ Õýæ©ÙèÁ Ú¹èÐ ÂýˆØð·UUUU ŒÜðÅ ÂÚ ç·UUUUÌÙð Õýæ©Ùè ãôÌð ãñ¢?	मिरांडा ने १८ ब्राउनीज को बेक किया। उसने २ प्लेटों पर समान सं2या में ब्राउनीज रखी। प्रत्येक प्लेट पर कितने ब्राउनी होते हैं?				9				
-			260002	word_problem	°·UUUU ç·UUUUâæÙ ·ðUUU Âæâ 16 ¥¢Çð ãñ¢ ¥æñÚ ßô ©‹ãð¢ â×æÙ MUUUUÂ âð 2 Õvâð ×ð¢ Ú¹Ìæ ãñÐ ÂýˆØð·UUUU Õæòvâ ×ð¢ ç·UUUUÌÙð ¥¢Çð ãñ¢?	एक किसान ने १६ अंडे इकट्ठे किए। नाश्ता बनाने के लिए उसने ४ अंडों का इस्तेमाल किया। उसके पास कितने अंडे बचे हैं?				8				
-			260003	word_problem	âæÚæ Ùð 6 ÈêUUUÜ ¿éÙðÐ ©âÙð ÈêUUUÜô¢ ·UUUè â×æÙ ×æ˜ææ ×ð¢ 2 ÈêUUUÜÎæÙô¢ ×ð¢ ÇæÜ ÎèÐ ÂýˆØð·UUUU ÈêUUUÜÎæÙ ×ð¢ ©âÙð ç·UUUUÌÙð ÈêUUUÜ ÇæÜð¢ ?	सारा ने ६ फूल चुने। उसने फूलों की समान मात्रा में २ फूलदानों में डाल दी। प्रत्येक फूलदान में उसने कितने फूल डालें ?				3				
-			260004	word_problem	àæðÙ Ùð °·UUUU ÂðÇ¸ âð 6 ¥æ× çÙ·UUUæÜðÐ ©‹ãô¢Ùð ¥æ×ô¢ ·UUUè °·UUUU â×æÙ â¢wØæ ·UUUUô 3 Õvâð ×ð¢ Ú¹æÐ ÂýˆØð·UUUU çÇyÕð ×ð¢ ©âÙð ç·UUUUÌÙð ¥æ× Ú¹ð?	शेन ने एक पेड़ से ६ आम निकाले। उन्होंने आमों की एक समान संख्या को ३ बक्से में रखा। प्रत्येक डिब्बे में उसने कितने आम रखे?				2				
-			260005	word_problem	20 Àæ˜æô¢ ·UUUUô çàæÿæ·UUUU mæÚæ 4 â×êãô¢ ×ð¢ â×æÙ MUUUUÂ âð çßÖæçÁÌ ç·UUUUØæ »Øæ ÍæÐ ÂýˆØð·UUUU â×êã ×ð¢ ç·UUUUÌÙð Àæ˜æ ãñ¢?	२० छात्रों को शिक्षक द्वारा ४ समूहों में समान रूप से विभाजित किया गया था। प्रत्येक समूह में कितने छात्र हैं?				5				
+			260001	word_problem	ç×ÚUæ´Çæ Ùð 18 Õýæ©Ùè ·¤ô Õð·¤ ç·¤ØæÐ ©âÙð 2 ŒÜðÅUô´ ÂÚU â×æÙ â´Øæ ×ð´ Õýæ©Ùè ÚU¹èÐ ÂýˆØð·¤ ŒÜðÅU ÂÚU ç·¤ÌÙð Õýæ©Ùè ãôÌð ãñ´?	मिरांडा ने १८ ब्राउनी को बेक किया। उसने २ प्लेटों पर समान संख्या में ब्राउनी रखी। प्रत्येक प्लेट पर कितने ब्राउनी होते हैं?				9				
+			260002	word_problem	°·UUUU ç·UUUUâæÙ ·ðUUU Âæâ 16 ¥¢Çð ãñ¢ ¥æñÚ ßô ©‹ãð¢ â×æÙ MUUUUÂ âð 2 Õvâð ×ð¢ Ú¹Ìæ ãñÐ ÂýˆØð·UUUU Õæòvâ ×ð¢ ç·UUUUÌÙð ¥¢Çð ãñ¢?	एक किसान के पास १६ अंडे हैं और वो उन्हें समान रूप से २ बक्से में रखता है। प्रत्येक बक्से में कितने अंडे हैं?				8				
+			260003	word_problem	âæÚUæ Ùð 6 È¤êÜ ¿éÙðÐ ©âÙð È¤êÜô´ ·¤ô â×æÙ ×æ˜ææ ×ð´ 2 È¤êÜÎæÙô´ ×ð´ ÇæÜ çÎØæÐ ÂýˆØð·¤ È¤êÜÎæÙ ×ð´ ©âÙð ç·¤ÌÙð È¤êÜ ÇæÜð´ ?	सारा ने ६ फूल चुने। उसने फूलों को समान मात्रा में २ फूलदानों में डाल दिया। प्रत्येक फूलदान में उसने कितने फूल डालें ?				3				
+			260004	word_problem	àæðÙ Ùð °·¤ ÂðÇ¸ âð 6 ¥æ× çÙ·¤æÜðÐ ©‹ãô´Ùð ¥æ×ô´ ·¤è °·¤ â×æÙ â´Øæ ·¤ô 3 Õ€Uâð ×ð´ ÚU¹æÐ ÂýˆØð·¤ Õ€Uâð ×ð´ ©âÙð ç·¤ÌÙð ¥æ× ÚU¹ð?	शेन ने एक पेड़ से ६ आम निकाले। उन्होंने आमों की एक समान संख्या को ३ बक्से में रखा। प्रत्येक बक्से में उसने कितने आम रखे?				2				
```

Comment:
`बक्से ` written as `बसे` in the Chanakya encoding.

Reply:
Resolved